docs/admins/Traffic-Light.md

```diff
@@ -0,0 +1,228 @@
+# Traffic Light Admin Guide
+
+For general end user info, see [users guide](/users/Projects/Traffic-Light/). Otherwise, for nitty gritty technical info, read on!
+
+## Acquiring
+
+Where did we get this? How do we get another if ours explodes? Exercise left to reader.
+
+## Lights
+
+Lights are regular 100w bulbs.  You can swing the front of the fixture open to replace them when they go out.
```

Comment:
@krux702  - is this right? I did want to publish it upfront and not wait 'til later.

Reply:
They are LED bulbs, not 100w

Reply:
Cool - can you recommend the text to put in here then? Maybe this?
> Lights are A19 LED 14w (100w equivalent)?